docs/sphinx/developers/commit-testing.txt

```diff
@@ -4,81 +4,107 @@ Testing code changes
 Automated tests
 ---------------
 
-At the bottom of many commit messages in
-https://github.com/openmicroscopy/bioformats, you will find a few lines
-similar to this:
+The :source:`Bio-Formats testing framework <components/test-suite>` component
+contains most of the infrastructure to run automated tests against the data
+repository.
 
-::
+After checking out source code and building all the JAR files (see
+:doc:`building-bioformats`), switch to the test-suite component and run the tests using the :program:`ant test-automated` target::
```

Comment:
Only `ant` should be marked up as a program.  test-automated should be verbatim only.